src/test/java/org/minions/devfund/angela/moviedatabase/MovieDatabaseTest.java

```diff
@@ -0,0 +1,93 @@
+package org.minions.devfund.angela.moviedatabase;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Test for {@link MovieDatabase}.
+ */
+public class MovieDatabaseTest {
+
+    private MovieDatabase movieDatabase;
+    private static final String MOVIE_NAME = "Infinity War";
+
+    /**
+     * Preconditions.
+     */
+    @Before
+    public void setup() {
+        movieDatabase = new MovieDatabase();
+        movieDatabase.addMovie(MOVIE_NAME, new String[]{"Robert Downey", "Chris Hemsworth", "Chris Evans"});
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Robert Downey" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)